tests/test_jwt.py

```diff
@@ -302,7 +306,7 @@ def test_before_request_one_time_token(self):
             mock.Mock(), 'GET', 'http://example.com?a=1#3', headers)
 
         header_value = headers['authorization']
-        token = header_value.split().pop()
+        token = header_value.split('Bearer ').pop()
```

Comment:
Still not happy here. Do the split on its own, then pop the token, then verify that `parts == []` after popping